### ISSUE TITLE
Tab item title is getting trimmed in various languages 

### ISSUE THREAD

**Issue by reeteshsingh93** [carbon-design-system/carbon#5783]
## Environment

> Operating system
Mac OS catalina

> Browser
Chorme Version 80.0.3987.149

## Detailed description

> What version of the Carbon Design System are you using?
  @console/pal: "1.100.2"

> What did you expect to happen?
Complete text should be visible to user 

> What happened instead?
Title text got trimmed

## Steps to reproduce the issue

1. Step one
Change borwser locale to french 
2. Step two
Open this URL : https://dev.console.test.cloud.ibm.com/security-advisor#/integrations

**Comment by asudoh** [comment]
Hi 👋 thank you for reporting - For this use case, our recommendation is overriding our style and set the widths that accommodates the most verbose language.

**Comment by asudoh** [comment]
Closing - Let us know if have any further questions.
